src/simply/simply_voice.c

```diff
@@ -0,0 +1,100 @@
+#include "simply_voice.h"
+
+#include "simply_msg.h"
+
+#include "simply.h"
+
+#include <pebble.h>
+
+typedef struct VoiceDataPacket VoiceDataPacket;
+
+struct __attribute__((__packed__)) VoiceDataPacket {
+  Packet packet;
+  int8_t status;
+  char result[SIMPLY_VOICE_BUFFER_LENGTH];
+};
+
+static SimplyVoice *s_voice;
+
+static bool send_voice_data(int status, char *transcription) {
+  VoiceDataPacket packet = {
+    .packet.type = CommandVoiceData,
+    .packet.length = sizeof(packet),
+    .status = (uint8_t) status,
+  };
+  snprintf(packet.result, sizeof(packet.result), "%s", transcription);
+
+  return simply_msg_send_packet(&packet.packet);
+}
+
+#ifndef PBL_SDK_2
+// Define a callback for the dictation session
+static void dictation_session_callback(DictationSession *session, DictationSessionStatus status, char *transcription, void *context) {
```

Comment:
This codebase has max line of 120 (although I'm starting to think about whether it should be 100) following Google C++ Style.

Long story short, you can format this like this.
```
static void dictation_session_callback(DictationSession *session, DictationSessionStatus status, char *transcription,
                                       void *context) {
```